HISTORY.md

```diff
@@ -1,5 +1,11 @@
 # History
 
+## v4.0.0 2020 June 19
```

Comment:
Said it is bumped to v4, but it is bumped to v3 inside `package.json` - seems v3 is what is intended here